boilerplate/scripts/webpack.config.babel.js

```diff
@@ -6,6 +6,9 @@ import pkg from "../package.json"
 import config from "./config.js"
 
 export default {
+  ...config.dev && {
+    devtool: 'cheap-module-eval-source-map',
```

Comment:
Ah. Because linter can auto fix this. 

This value is recommended by my team leader. It just work for me.